mobile/lib/modules/backup/background_service/background.service.dart

```diff
@@ -389,8 +378,8 @@ class BackgroundService {
         return false;
       }
       // check for new assets added while performing backup
-    } while (true ==
-        await _backgroundChannel.invokeMethod<bool>("hasContentChanged"));
+    } while (false); //TODO: broken on iOS (true ==
```

Comment:
This needs some better treatment for iOS, as we do not check for new assets added while performing backup in iOS.